pom.xml

```diff
@@ -258,6 +258,21 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <!-- This defines the properties 'parsedVersion.majorVersion'
+                    and 'parsedVersion.minorVersion' to be in synch with the current project version -->
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.9.1</version>
```

Comment:
Could you put the version into `pluginManagement`? That's how we did it for all other plug-ins.